src/regression.jl

```diff
@@ -0,0 +1,33 @@
+function conjugate(r::Regression, Y, Ξ)
+  @unpack k, fjac!, ϕ0func!, ϕ, μ, Γ, ϕ0, y, y2 = r
+  @unpack g, pest = k
+
+  t = Y.t[1]
+  y .= Y.u[1]
+
+  fjac!(ϕ, y, pest, t)
+  fill!(μ, zero(eltype(ϕ)))
```

Comment:
Let's allow `μ` and `Γ` arguments to `conjugate`, so several segments can be chained.

Reply:
Added them as:

```.. , μ0=nothing, Γ0=nothing)```

so if not specified they will be filled with `zero` , if specified the values are written into `μ` and `Γ`.